--- a/src/daemon/systemd/system/meson.build
+++ b/src/daemon/systemd/system/meson.build
@@ -1,4 +1,4 @@
-systemd_system_services_dir = systemd.get_variable(pkgconfig: 'systemdsystemunitdir', pkgconfig_define : [ 'rootprefix', prefix])
+systemd_system_services_dir = systemd.get_variable(pkgconfig: 'systemdsystemunitdir')
 
 install_data(sources : 'pipewire.socket',
              install_dir : systemd_system_services_dir)
